monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -144,8 +144,28 @@ class ConfigurePageComponent extends AuthComponent {
       });
   }
 
-  updateMasqueStrings = () => {
-    console.log('MASQUE update');
+  updateMasqueStrings = async () => {
+    const [linuxRes, windowsRes] = await Promise.all([
+      this.authFetch('/api/agent-binaries/linux/masque', {}, true),
+      this.authFetch('/api/agent-binaries/windows/masque', {}, true)
+    ]);
+    const linuxMasqueBytes = await linuxRes.arrayBuffer();
+    const linuxMasqueString = this.getStringFromBytes(linuxMasqueBytes);
+
+    const windowsMasqueBytes = await windowsRes.arrayBuffer();
+    const windowsMasqueString = this.getStringFromBytes(windowsMasqueBytes);
```

Comment:
Use `IslandHttpClient` for API calls.

Reply:
I will need to add a new `get` as we are not retrieving json but `IslandHttpClient.get` response tries to get json.

Reply:
Let's do that then. API calls should be isolated from the rest of the application rather than scattering them throughout the codebase.